src/absil/il.fs

```diff
@@ -3699,7 +3708,11 @@ type ILTypeSigParser(tstring : string) =
                       yield grabScopeComponent() // culture
                       yield grabScopeComponent() // public key token
                     ] |> String.concat ","
+#if FABLE_COMPILER
```

Comment:
Please use ``FX_NO_ASSEMBLYNAME`` or add an adapter for the AssemblyName type, thanks